### ISSUE TITLE
[Accessibility]: Multiple H1 tags on Code monitor getting started page

### ISSUE THREAD

**Issue by benvenker** [sourcegraph/sourcegraph#35154]
### Audit type

ARC Toolkit

### Problem description

There are two H1s on the Code monitoring getting started page

### Expected behavior

There is only one H1 tag

### Additional details

![image](https://user-images.githubusercontent.com/4605817/167486143-33f4a4d8-82e3-4786-af5c-bf06a485c5ef.png)

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.

**Comment by gitstart-sourcegraph** [comment]
Hi @limitedmage 
Looks like this issue has been resolved as part of previous efforts, kindly help to confirm if you are still able to reproduce this.
CC: @benvenker

**Comment by limitedmage** [comment]
Thanks @gitstart-sourcegraph, this looks good now!
